crates/timers/src/future.rs

```diff
@@ -1,8 +1,11 @@
 //! `Future`- and `Stream`-backed timers APIs.
 
+use std::task::{Poll, Context};
+use std::pin::Pin;
+use std::future::Future;
 use super::sys::*;
-use futures::prelude::*;
-use futures::sync::mpsc;
+use futures::stream::Stream;
```

Comment:
We should be using individual crates like `futures_core` rather than the blanket `futures`.